boot/freeldr/fdebug/lang/ro-RO.rc

```diff
@@ -50,36 +56,36 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
+    LTEXT "Treceți portul COM (ex. COM1):", IDC_STATIC, 7, 7, 108, 8
     EDITTEXT IDC_COMPORT, 7, 17, 182, 14, ES_AUTOHSCROLL
-    LTEXT "Rata baud (ex. 115200):", IDC_STATIC, 7, 38, 114, 8
+    LTEXT "Treceți rata baud (ex. 115200):", IDC_STATIC, 7, 38, 114, 8
     EDITTEXT IDC_BAUTRATE, 7, 48, 182, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", IDOK, 45, 79, 50, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 100, 79, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 45, 79, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 100, 79, 50, 14
 END
 
 IDD_CAPTURE DIALOGEX 0, 0, 251, 95
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Fișierul de captură"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Numele fișierului:", IDC_STATIC, 7, 17, 62, 8
+    LTEXT "Nume fișier capt.:", IDC_STATIC, 7, 17, 62, 8
     EDITTEXT IDC_CAPTUREFILENAME, 7, 26, 181, 14, ES_AUTOHSCROLL
-    PUSHBUTTON "&Caută", IDC_BROWSE, 194, 26, 50, 14
-    DEFPUSHBUTTON "Con&firmă", IDOK, 139, 74, 50, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 194, 74, 50, 14
+    PUSHBUTTON "&Specificare", IDC_BROWSE, 194, 26, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 139, 74, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 194, 74, 50, 14
 END
 
 /* String Tables */
 
 STRINGTABLE
 BEGIN
-    IDS_APP_TITLE "Program de depanare FreeLoader"
-    IDS_HELLO "Salut, lume!"
+    IDS_APP_TITLE "Depanatorul FreeLoader"
+    IDS_HELLO "Salut lume!"
     IDC_FDEBUG "FDEBUG"
 END
 
 STRINGTABLE
 BEGIN
-    IDS_LICENSE "Această aplicație este un soft public; poate fi redistribuită și/sau modificată respectând termenii Licenței Publice Generale GNU publicată de Free Software Foundation; sau respectând Licența în versiunea 2, sau (la alegere) o altă versiune ulterioară.\r\n\r\nAceastă aplicație este distribuită în speranța că va fi utilă, FĂRĂ însă NICI O GARANŢIE; nici măcar cu garanția implicită a VANDABILITĂŢII sau a UTILITĂŢII ÎNTR-UN SCOP ANUME.  Consultați Licența Publică Generală GNU pentru mai multe detalii.\r\n\r\nPuteți vedea această licență aici:\r\nhttp://www.gnu.org/licenses/gpl.html\r\n\r\nDe asemenea puteți consulta traduceri neoficiale ale acestei licențe aici:\r\nhttp://www.gnu.org/licenses/translations.html"
+    IDS_LICENSE "Acest program este software gratuit; îl puteți redistribui și/sau modifica în conformitate cu termenii Licenței Publice Generale GNU publicate de Free Software Foundation; fie versiunea 2 a Licenței, fie (la alegerea dvs.) orice versiune ulterioară.\r\n\r\nAcest program este distribuit în speranța că va fi util, dar FĂRĂ NICIO GARANȚIE; fără nici măcar garanția implicită de VANTABILITATE sau ADECVARE PENTRU UN ANUMIT SCOP. Consultați Licența publică generală GNU pentru mai multe detalii.\r\n\r\nAr fi trebuit să primiți o copie a Licenței publice generale GNU împreună cu acest program; dacă nu, scrieți la Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 SUA."
 END
```

Comment:
Question to others: How can we make ros display that string? Is it really needed to use `\r\n` here instead of just `\n` like we can do it in 99% of other rc's?

Reply:
I don't know. I took the text from English in order to retranslate the string. I can change from \r\n to \n if you want.